### ISSUE TITLE
Cache time-dynamic raster tiles for smoother playback

### ISSUE THREAD

**Issue by rsignell-usgs** [TerriaJS/terriajs#1451]
This feature request is related to #984.   

It would be great to be able to have images in a time loop cached for rapid playback when the loop repeats (this is what the Godiva2/Godiva3 WMS clients do).

It would dramatically help time animations like: 
http://comt.sura.org/proxy_3001/#https://raw.githubusercontent.com/rsignell-usgs/ehzrap/master/testing/bird_migration.json
where the same 12 monthly WMS layers of air temperatures get requested over and over, but can't keep up with the bird populations.

**Comment by kring** [comment]
FWIW, while TerriaJS/Cesium won't cache looping images, the browser should and does.  On my system, at least, the second play through is pretty smooth.

It would be interesting to cache the loaded images in TerriaJS/Cesium, too, rather than relying on the browser cache.  I think we would want to structure it as a cache of multiple time-based images of a certain tile so that we can evict images for tiles that are no longer in view at all.  This likely need to be a Cesium change; I can't think of how to do it within TerriaJS.

**Comment by kring** [comment]
I should mention that the advantage of caching loaded tiles is it requires us to do less work to show those images again.  On slower systems that could make a big difference in the smoothness of the animation.

**Comment by rsignell-usgs** [comment]
@kring, I guess you are talking about this?
https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching#cache-control

**Comment by kring** [comment]
No, I think the HTTP headers are already fine (though it might be worth spending a minute to make sure).   Talking about keeping the map images loaded within our engine, E.g. as WebGL textures rather and going through the process of loading them (from browser cache) when they're needed.
